python/paddle/fluid/contrib/utils/hdfs_utils.py

```diff
@@ -475,12 +428,82 @@ def __subprocess_download(datas):
     for data in need_download:
         data_name = os.path.basename(data)
         re_path = os.path.relpath(os.path.dirname(data), hdfs_path)
-        local_re_path = os.path.join(local_path, re_path, data_name)
+        if re_path == os.curdir:
+            local_re_path = os.path.join(local_path, data_name)
+        else:
+            local_re_path = os.path.join(local_path, re_path, data_name)
         local_downloads.append(local_re_path)
 
     return local_downloads
 
 
+def getfilelist(path):
+    rlist = []
+    for dir, folder, file in os.walk(path):
+        for i in file:
+            t = os.path.join(dir, i)
+            rlist.append(t)
+    for r in rlist:
+        print(r)
+
+
+def multi_upload(client,
+                 hdfs_path,
+                 local_path,
+                 multi_processes=5,
+                 overwrite=False,
+                 sync=True):
+    """
+
+    :param client:
```

Comment:
Docstring style and need finish documents.

Reply:
Done.